OneOneOne/OneOneOne/message_handler.py

```diff
@@ -0,0 +1,190 @@
+from lxml import etree, objectify
+import xml.etree.ElementTree as ET
+
+from builder.pystache_message_builder import PystacheMessageBuilder
+from utilities.file_utilities import FileUtilities
+from definitions import XML_PATH, TEMPLATE_PATH
+import logging
+
+basic_success_response = FileUtilities.get_file_string(XML_PATH / 'basic_success_response.xml')
+
+base_fault_response = FileUtilities.get_file_string(XML_PATH / 'basic_fault_response.xml')
```

Comment:
should that be `basic_fault_response`?

Reply:
Yeah that makes sense for them to be the same, updated.

Reply:
Why not just use a `build_error_message("Invalid message")` call instead of having this one error response defined in its own file?

Reply:
Good point, I've removed the `basic_fault_response`